.github/actions/prepare-install/action.yml

```diff
@@ -55,6 +55,6 @@ runs:
       shell: bash
       working-directory: ${{ inputs.working-directory }}
       run: |
-        yarn --ignore-engines --frozen-lockfile --ignore-scripts
```

Comment:
https://github.com/typescript-eslint/typescript-eslint/actions/runs/5207310238/jobs/9394739015

```plaintext
warning Lockfile has incorrect entry for "typescript@>=3.3.1 <5.2.0". Ignoring it.
error Your lockfile needs to be updated, but yarn was run with `--frozen-lockfile`.
```

I don't get this locally 😡